source/index.html.erb

```diff
@@ -143,13 +143,23 @@
     </div>
     <div class="content-section content-section--with-top-border">
       <div class="grid-row">
-          <div class="column-full">
-            <h2 class="content-section__title heading-medium">GOV.UK&nbsp;Pay is already working with:</h2>
-            <img class="content-section__image partner-logo" src="/pay-product-page/images/ministry-of-justice.png" alt="Logo of the Ministry of Justice" />
-            <img class="content-section__image partner-logo" src="/pay-product-page/images/home-office.png" alt="Logo of the Home Ofiice" />
-            <img class="content-section__image partner-logo" src="/pay-product-page/images/beis.png" alt="Logo of the Department for Business, Energy &amp; Industrial Strategy" />
-            <img class="content-section__image partner-logo" src="/pay-product-page/images/office-public-guardian.png" alt="Logo of the Office of the Public Guardian" />
+        <div class="column-full">
+          <h2 class="content-section__title heading-medium">GOV.UK&nbsp;Pay is already working with:</h2>
+          <div class="cf">
+            <img class="content-section__image partner-logo" src="/pay-product-page/images/logos/moj.png" alt="Logo of the Ministry of Justice" />
```

Comment:
I think we should drop “Logo of” from all the alt texts.

The textual equivalent is the names of the organisations; the fact that they’re logos when presented visually isn’t important.

If we just didn’t use logos and had a purely textual list, we’d just put the names of the organisations, so we should do the same with the alt text.